### ISSUE TITLE
EKS and possibly other installers need to account for new org/project/stack naming convention used for self-managed backends

### ISSUE THREAD

**Issue by MitchellGerdisch** [pulumi/pulumi-self-hosted-installers#94]
### What happened?

Pulumi 3.61.0 (ish) introduced `org/project/stack` type naming conventions for stacks in self-managed backends.
This is causing the EKS installer to fail since, say, stack `02-cluster-services` is looking for a stack named, say, `01-cluster-configuration` but the system is returning `organization/01-cluster-configuration/01-cluster-configuration` and so the installer failes.
The other installers probably have the same problem - haven't tested yet to confirm.

**WORKAROUND** 
Set environment variable:
`PULUMI_SELF_MANAGED_STATE_LEGACY_LAYOUT=1`

Specifically when deploying a new self-hosted installation do the following to avoid the issue:

- CREATE a new self-managed backend for the self-hosted installation.
- BEFORE running any of the installers or `pulumi` commands, `export PULUMI_SELF_MANAGED_STATE_LEGACY_LAYOUT=1`

NOTE: If you ran into this problem, the backend is tainted and you'll need to clean up the resources, and then start afresh with a new backend.

### Expected Behavior

Things need to work with the new naming convention

### Steps to reproduce

Try launching an eks cluster using a new s3 bucket backend.
(NOTE: pre-existing self-managed backends will work since the legacy naming convention will continue to be used.)

### Output of `pulumi about`

stuff

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by rshade** [comment]
Currently, all installer READMEs contain references to the legacy self-managed stack names (eg- my-stack). Notice there is not fully qualified stack name. With, the changes listed [here](https://www.pulumi.com/blog/project-scoped-stacks-in-self-managed-backend/) self-managed backends now contain fully qualified stack names.

Project scoped stacks take effect automatically with new state backends (empty buckets, blobs, etc).
For existing state backends who want to upgrade, see https://www.pulumi.com/blog/project-scoped-stacks-in-self-managed-backend/#upgrading-existing-backends.
available in v3.61.0 (pu/pu)

**Comment by cleverguy25** [comment]
@rshade @jdavredbeard Can we close this now?

**Comment by rshade** [comment]
I don't think we actually updated all the stacks to match the full stack names. I think probably just AWS
